README.md

```diff
@@ -1,4 +1,48 @@
 # ibm-watson-embed-model-builder
 
 This python library manages the process of building a collection of docker images that wrap individual watson_embedded 
-models for delivery both internal and external to IBM.
+models for delivery with an embeddable watson runtime.
+
+## Overview
+
+Given a set of `watson_embed` models to be packaged, this tool can create a model manifest file describing the model
+images and metadata to be created.
+
+Given a model manifest file, this tool can fetch the model artifacts and package them into watson runtime compatible 
+model images along with useful metadata.
+
+These operations are split into separate commands to facilitate easy parallelization in the CI system of your choice.
+
+## Installation
+
+```shell
+pip install watson_embed_model_packager
+```
+
+## Usage
+
+To build a model manifest file from models hosted in an artifactory instance, you will need:
+
+- A list of all module GUIDs to support
```

Comment:
nit: this list has 2 capitalized letters, and 3 non-capitalized letters lol. Let's pick one of the other!